vmdb/app/models/ems_microsoft.rb

```diff
@@ -96,7 +96,8 @@ def execute_power_operation(cmdlet, vm_uid_ems, *parameters)
     return unless vm_uid_ems.guid?
 
     params  = parameters.join(" ")
-    command = "powershell #{cmdlet}-SCVirtualMachine -VM (Get-SCVirtualMachine -ID #{vm_uid_ems}) #{params}"
+    command = "powershell Import-Module VirtualMachineManager; Get-VMMServer #{ipaddress}; "
```

Comment:
Great @bronaghs !  I'm :+1: for hardcoding "localhost" in this PR and adding a #TODO so we don't lose the knowledge that we need to wrap ipv6 addresses when using Get-VMMServer.  Someday, SCVMM may be running on an IPv6 only network and we may need to use `::1`.